cypress.config.js

```diff
@@ -1,8 +1,32 @@
 const { defineConfig } = require('cypress')
-
+const faker = require('faker');
 module.exports = defineConfig({
   e2e: {
+    viewportHeight: 1080,
+    viewportWidth: 1920,
     setupNodeEvents(on, config) {
+      on("task",{
+        generateUser() {
+          genders = ['Female', 'Male', 'Other'];
+          subjects = ['Commerce', 'Civics', 'Biology'];
+          hobbies = ['Music', 'Sports', 'Reading'];
```

Comment:
Declare the arrays 'genders', 'subjects', and 'hobbies' as constants outside of the 'generateUser' function to avoid redeclaring them each time the function runs.

Reply:
done.please check is it correct?